src/components/Popper/Popper.tsx

```diff
@@ -73,35 +73,27 @@ export const Popper = ({
                     const { name } = child.type;
 
                     if (name === Reference.name) {
-                        return (
-                            <div className="tw-w-fit" ref={setReferenceElement}>
-                                {cloneElement(child, { ...child.props })}
-                            </div>
-                        );
+                        return <div ref={setReferenceElement}>{child}</div>;
                     }
 
                     if (name === Content.name) {
                         return enablePortal ? (
                             <Portal>
                                 <div
                                     ref={setPopperElement}
-                                    style={popperInstance.styles.popper}
+                                    style={{ zIndex: 9999, ...popperInstance.styles.popper }}
```

Comment:
Quite a bad for scaling, please create variables for the different cases you have.
Examples:

Best read: [Chakra UI](https://chakra-ui.com/community/recipes/z-index)
Alternatives:
- [MUI](https://mui.com/material-ui/customization/z-index/)
- [Bootstrap](https://getbootstrap.com/docs/5.0/layout/z-index/)
- [CoreUI](https://coreui.io/docs/layout/z-index/)

Reply:
Refactored! Thanks for pointing out. :)